### PR TITLE
docs: minimally outline local labels

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

File: docs/labels.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- vm
+- wallet
+
+## Platform
+
+In general we support Linux, Mac and Windows, but currently we use this:
+
+- windows
+  
+  It caused so many problems that it deserves a label of its own.
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply docs/labels.md:23]: It's intentional, to be a paragraph for `windows`, like in the original `label.md`.

File: docs/labels.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- cli
+- compiler
+- consensus
+- network
+- oracle
+- rpc
+- smartcontract
+- vm
+- wallet
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply docs/labels.md:15]: Likely so.

File: docs/labels.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+There is a number of internal components:
+
+- cli
+- compiler
+- consensus
+- network
+- oracle
+- rpc
+- smartcontract
+- vm
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply docs/labels.md:14]: Fixed.

File: docs/labels.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (roman-khimov) [reply docs/labels.md:15]: Although we don't have a lot of issues at the moment. This set was taken from existing ones.

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**